etc/test_config/sdk/test-vectors.json

```diff
@@ -45,11 +45,11 @@
         "outputs": {
           "signBytes": "0xfa010000002cede35562d3555e61120a151b3c8e8e91d83a378a19aa2ed8712072e918632259780e587698ef58df000000004a817c80027d030000000c000000000000000000000000ffffffff",
           "signature": {
-            "pubKey": "40771354dc314593e071eaf4d0f42ccb1fad6c7006c57464feeb7ab5872b7490",
-            "signature": "b3211c7e15d31d64619e0c7f65fce8c6e45637b5cfc8711478c5a151e6568d875ec7f48e040225fe3cc7f1e7294625cad6d98b4595d007d36ef62122de16ae01"
+            "pubKey": "9fce9db2059996381a66dcfbadca9f31629c04d894c4cefe137b2d29c95a5793",
+            "signature": "e9b5e8de5edd93310b8a9e874436435109f2ca96d8a10b05759c05fc40ebca96551404d14b8bfdb02045e52fe4a4575011a09dfd8e46eefe1b18f47aa4526e01"
           },
-          "ethSignMessage": "0x5472616e7366657220313030303030303030303030302e302045544820746f3a203078313961613265643837313230373265393138363332323539373830653538373639386566353864660a4665653a20313030303030302e30204554480a4e6f6e63653a203132",
-          "ethSignature": "0x4684a8f03c5da84676ff4eae89984f20057ce288b3a072605cbf93ef4bcc8a021306b13a88c6d3adc68347f4b68b1cbdf967861005e934afa50ce2e0c5bced791b"
+          "ethSignMessage": "0x5472616e7366657220313030303030303030303030302e30205242544320746f3a203078313961613265643837313230373265393138363332323539373830653538373639386566353864660a4665653a20313030303030302e3020524254430a4e6f6e63653a203132",
+          "ethSignature": "0x31667c153fd6fc477c2c5ee518ca247054b197d690dd5b5e53a5bb5ce39ddde5529f44141f68c7124c99cf17e1464255b0a76877726d6a8ee18c01694eba9c131b"
```

Comment:
why did all these hashes change? Do the orig messages contain the changed word rollup?

Reply:
yes, since we changed contracts and those contracts generated signatures, it was necessary to change it.